GameServerLib/Config.cs

```diff
@@ -256,6 +256,26 @@ public LaneID GetLaneID()
                 return laneId;
             }
 
+            public LaneID GetSpawBarrackLaneID()
```

Comment:
TYPO: Should be `Spawn` instead of `spaw`.

Reply:
Resolved